docs/data/joy/main-features/global-variants/global-variants.md

```diff
@@ -1,34 +1,36 @@
 # Global variants
 
-<p class="description">Joy UI enables consistent look and feel across all components by using global variants.</p>
+<p class="description">Joy UI provides a set of global variants to ensure consistency across your app.</p>
 
-Every Joy UI component has the same set of 4 global variants.
-All of them are pulling styles from the same place, ensuring then that each variant looks the same across both "native" and custom components.
-That way, you get a consistent look and feel throughout your design system.
+All Joy UI components accept four global variants: `solid` (the default), `soft`, `outlined`, and `plain`. These variants are intended to cover the majority of use cases in modern web design.
```

Comment:
```suggestion
All Joy UI components accept four global variants: `solid`, `soft`, `outlined`, and `plain`. These variants are intended to cover the majority of use cases in modern web design.
```

The default variant depends on the type of the components. For example, the Button is an call-to-action component, so it has `solid` as a default variant. The form controls, e.g. Input, has `outlined` as a default variant.

Reply:
Thanks for the clarification!